frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/ActionOptions/ActionOptions.tsx

```diff
@@ -57,6 +59,12 @@ function ActionOptions({
     (action: WritebackAction) => {
       onUpdateButtonActionMapping(dashcard.id, {
         action_id: action.id,
+        action,
+        visualization_settings: {
+          ...dashcard.visualization_settings,
+          "button.label":
```

Comment:
Not sure if this logic should be done within our settings framework https://github.com/metabase/metabase/tree/5a0a16dd11a6538196cf8acf3e3903d4bc8e4458/frontend/src/metabase/visualizations/lib/settings